### PR TITLE
Quiz app

### PR DESCRIPTION
**Pull request by StephanLippert**
I Changed a few things....and a lot is still missing 😅

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -2,3 +2,5 @@
 @import url(components/header.css);
 @import url(components/section.css);
 @import url(components/footer.css);
+@import url(form.css);
+@import url(profil.css);
```
Comment (NikRoe) [inline styles.css:6]: your file is called `profile.css` so this import currently doesn't work;

```suggestion
@import url(profile.css);
```

File: form.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          <label for="yourTag">Your Tag:</label>
+          <input
+            type="text"
+            name="yourTag"
+            id="yourTag"
+            placeholder="enter your Tag here..."
+            required
+          ></input>
+        </fieldset>
+        <button class="submitButton">submit</button>
```
Comment (NikRoe) [inline form.html:59]: try to be explicit about the types of your buttons:

```suggestion
        <button class="submitButton" type="submit">submit</button>
```

File: form.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      >Your Q&A's
+    </h1>
+    </header>
+  <main>
+      <form data-js="form" class="form">
+        <fieldset>
+          <legend>Questions & Answers</legend>
+          <label for="yourQuestionArea">Your Question:</label>
+          <textarea
+            type="text"
```
Comment (NikRoe) [inline form.html:24]: you don't need this attribute, since a textarea cannot have different types

```suggestion
```

File: form.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            cols="40"
+            rows="4"
+            placeholder="enter your Question here..."
+            required
+          ></textarea>
+          <small class="form__character-count">
+            (<span data-js="amountLeftQuestion"></span> characters left)</small>
+          <label for="yourAnswerArea">The Answer:</label>
+          <textarea
+            type="text"
```
Comment (NikRoe) [inline form.html:38]:
```suggestion
```

File: form.html [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  <script src="form.js" defer></script>
+  <title>Form</title>
+</head>
+<body>
+  <body data-js="body">
+    <header class="header">
+    <h1
+      class="header__headline"
+      >Your Q&A's
+    </h1>
```
Comment (NikRoe) [inline form.html:16]: your Prettier appears not to work properly for this part of the code.
Does it format your code properly when you save a file?

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi Stephan,

this looks like a promising start!

I noticed a bug for one of your imports and left you a few comments :) 

Weiter so 👏

File: form.html (hunk 5 above)
Comment (StephanLippert) [reply form.html:16]: Yes, it usually does...but my site is kind of special...for example:
I've bild in two body_opening_tags...and in think there are some more Mistakes i've made...🤣
Thank you very much for taking a look 🙏🏼 I'll try to fix everything.😇👍🏼

**Pull request merged**